MarkBind/markbind
# Support `$$` as a delimiter for math formulae

**What is the purpose of this pull request?**




**Overview of changes:**
Fixes #1754

The inclusion of this optional plugin will allow authors who are using dollars as delimiters to adopt MarkBind easily and continue to display their tex Math equations in the same syntax.

**Anything you'd like to highlight / discuss:**
In UG I used the default math syntax to render the `$$` formulae because this plugin is not enabled for our docs.

**Testing instructions:**
See the preview page [here](https://deploy-preview-1824--markbind-master.netlify.app/userguide/formattingcontents#math-formulae) and [here](https://deploy-preview-1824--markbind-master.netlify.app/userguide/usingplugins#plugin-texwithdollars)
- some discussion on the approach is in the original issue #1754, read it for context

**Proposed commit message: (wrap lines at 72 characters)**
Support `$$` as a delimiter for math formulae

Current delimiters for math formulae do not allow authors
who are using dollars(`$$`) for their equations to adopt
MarkBind easily.

Let's add an optional plugin that can add `$$` as additional
delimiters for authors who have content with this syntax.

Since not all authors use MarkBind with math formulae, an
optional plugin can prevent any accidental performance
issues.



---

**Checklist:** :ballot_box_with_check:



fixed a very minor typo in the math formulae section
